script/install_boost.sh

```diff
@@ -145,7 +144,7 @@ then
     then
         ALPAKA_BOOST_B2_CXXFLAGS+=" -std=c++17"
     fi
-    ALPAKA_BOOST_B2+=" --with-fiber --with-context --with-thread --with-atomic --with-system --with-chrono --with-date_time"
+    ALPAKA_BOOST_B2+=" --with-context --with-thread --with-atomic --with-system --with-chrono --with-date_time"
```

Comment:
```suggestion
    ALPAKA_BOOST_B2+="--with-atomic"
```
Looking at https://pdimov.github.io/boostdep-report/boost-1.70.0/atomic.html, no built dependencies should be needed for Boost.Atomic.